app/code/Magento/SendFriendGraphQl/Model/Resolver/SendEmailToFriend.php

```diff
@@ -45,22 +47,30 @@ class SendEmailToFriend implements ResolverInterface
      */
     private $eventManager;
 
+    /**
+     * @var Visibility
+     */
+    private $visibility;
+
     /**
      * @param SendFriendFactory $sendFriendFactory
      * @param ProductRepositoryInterface $productRepository
      * @param DataObjectFactory $dataObjectFactory
      * @param ManagerInterface $eventManager
+     * @param Visibility|null $visibility
      */
     public function __construct(
         SendFriendFactory $sendFriendFactory,
         ProductRepositoryInterface $productRepository,
         DataObjectFactory $dataObjectFactory,
-        ManagerInterface $eventManager
+        ManagerInterface $eventManager,
+        ?Visibility $visibility= null
```

Comment:
Backward compatibility should not be followed in GraphQL php codebase - only schema counted as a contract.
https://github.com/magento/graphql-ce/wiki/Approach-to-Implementation#backward-compatibility-policy